lte/protos/mconfig/mconfigs.proto

```diff
@@ -71,6 +71,14 @@ message EnodebD {
         int32 tac = 8;
         int32 cell_id = 9;
         string ip_address = 10;
+        PowerControl power_control = 34;
+        bool x2_enable_disable = 112;
+        message PowerControl {
+          int32 reference_signal_power = 1;
+          uint32 power_class = 2;
+          uint32 pb = 3;
+          int32 pa = 4;
+        }
```

Comment:
Add some more detailed info on these definitions - anybody changing code relying on these definitions could find use in any linked specifications, or value limits to these power control parameters.